### ISSUE TITLE
how to use the tensorboard to visualize the model?

### ISSUE THREAD

**Issue by runzeer** [open-mmlab/mmdetection#623]

**Comment by hellock** [comment]
We will support it after an upstream bug being fixed. https://github.com/pytorch/pytorch/pull/20116

**Comment by yxchng** [comment]
@hellock In the mean time, is there any quick solution to use the `torch.utils.tensorboard` officially supported in PyTorch 1.1.0? Actually I just need a way to access the loss during training, then I would be able to code the rest myself.

**Comment by hellock** [comment]
@yxchng If you just want to log the loss, you can simply uncomment [this line](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn_r50_fpn_1x.py#L153).

**Comment by Lucy0601** [comment]
I want to visualize the model using TensorboardX too, but encountered with some problems, when will you support it?

**Comment by tommyjiang** [comment]
Hope this feature added soon, thank you.

**Comment by korabelnikov** [comment]
Hi, @hellock 
Is this answer still correct for new versions of mmdet?

I use
```
mmcv                   1.2.2               /home/user/repos/mmcv
mmcv-full              1.2.2               /home/user/repos/mmcv
mmdet                  2.7.0               /home/user/repos/mmdetection_torch1.6
tensorboard            2.4.1
tensorboard-plugin-wit 1.8.0
```
I've uncommented TensorBoardHook, executed dist_train but get lr, momentum, and mAP only
![image](https://user-images.githubusercontent.com/3808833/108190785-74110a80-7123-11eb-830c-a6c34b7dde0f.png)

**Comment by lushaoan** [comment]
> Hi, @hellock
> Is this answer still correct for new versions of mmdet?
> 
> I use
> 
> ```
> mmcv                   1.2.2               /home/user/repos/mmcv
> mmcv-full              1.2.2               /home/user/repos/mmcv
> mmdet                  2.7.0               /home/user/repos/mmdetection_torch1.6
> tensorboard            2.4.1
> tensorboard-plugin-wit 1.8.0
> ```
> 
> I've uncommented TensorBoardHook, executed dist_train but get lr, momentum, and mAP only
> ![image](https://user-images.githubusercontent.com/3808833/108190785-74110a80-7123-11eb-830c-a6c34b7dde0f.png)

I get the same problem, do you figure it out ?

**Comment by hhaAndroid** [comment]
Is there still a problem with the latest version now?

**Comment by korabelnikov** [comment]
it works once upgraded

**Comment by invincible-28** [comment]
> Hi, @hellock Is this answer still correct for new versions of mmdet?
> 
> I use
> 
> ```
> mmcv                   1.2.2               /home/user/repos/mmcv
> mmcv-full              1.2.2               /home/user/repos/mmcv
> mmdet                  2.7.0               /home/user/repos/mmdetection_torch1.6
> tensorboard            2.4.1
> tensorboard-plugin-wit 1.8.0
> ```
> 
> I've uncommented TensorBoardHook, executed dist_train but get lr, momentum, and mAP only ![image](https://user-images.githubusercontent.com/3808833/108190785-74110a80-7123-11eb-830c-a6c34b7dde0f.png)

Could you please share where did you find this dist_train file to visualize the results on tensorboard?

**Comment by roshambo919** [comment]
The link provided by @hellock is no longer valid. Uncommeting the `TensorboardLoggerHook` line in `configs/models/default_runtime.py` should do it in most cases.
